interceptors/logging/interceptors_test.go

```diff
@@ -463,28 +493,34 @@ func (s *loggingCustomDeciderSuite) TestPingError_HasCustomDecider() {
 	require.Error(s.T(), err, "each call here must return an error")
 
 	lines := s.logger.Lines()
-	require.Len(s.T(), lines, 2)
+	require.Len(s.T(), lines, 3)
 
-	serverLogLine := lines[0]
+	serverLogLine := lines[1]
 	assert.Equal(s.T(), logging.INFO, serverLogLine.lvl)
-	assert.Equal(s.T(), "finished server unary call", serverLogLine.msg)
+	assert.Equal(s.T(), "finished call", serverLogLine.msg)
 	serverFields := assertStandardFields(s.T(), logging.KindServerFieldValue, serverLogLine.fields, "PingError", interceptors.Unary)
 	serverFields.AssertNextField(s.T(), "grpc.request.value", "something").
 		AssertNextFieldNotEmpty(s.T(), "peer.address").
+		AssertNextField(s.T(), "protocol", "grpc").
+		AssertNextField(s.T(), "component", "server").
+		AssertNextField(s.T(), "type", "unary").
 		AssertNextFieldNotEmpty(s.T(), "grpc.start_time").
 		AssertNextFieldNotEmpty(s.T(), "grpc.request.deadline").
 		AssertNextField(s.T(), "grpc.code", "NotFound").
-		AssertNextField(s.T(), "error", "rpc error: code = NotFound desc = Userspace error.").
+		AssertNextField(s.T(), "grpc.error", "rpc error: code = NotFound desc = Userspace error.").
 		AssertNextFieldNotEmpty(s.T(), "grpc.time_ms").AssertNoMoreTags(s.T())
 
-	clientLogLine := lines[1]
+	clientLogLine := lines[2]
 	assert.Equal(s.T(), logging.DEBUG, clientLogLine.lvl)
-	assert.Equal(s.T(), "finished client unary call", clientLogLine.msg)
+	assert.Equal(s.T(), "finished call", clientLogLine.msg)
 	clientFields := assertStandardFields(s.T(), logging.KindClientFieldValue, clientLogLine.fields, "PingError", interceptors.Unary)
 	clientFields.AssertNextFieldNotEmpty(s.T(), "grpc.start_time").
 		AssertNextFieldNotEmpty(s.T(), "grpc.request.deadline").
 		AssertNextField(s.T(), "grpc.code", "NotFound").
```

Comment:
Reuse gives us power to reduce cognitive load which is resulted by complexity. And usually more lines of code means more stuff to read = complexity.

Thanks to reuse we did, we could .... reuse tests as well! So 5x less.

Now with our idea to have single in our code `addMeta` or `logStarted(` or even `logCall(isStarted, isSever)`, so we can focus on decider not on CHECKING this all the time: 
![image](https://user-images.githubusercontent.com/6950331/88287720-986e4b00-ccea-11ea-9126-b8a42f97b984.png)

So again, on top 5x we 3x less to test. We just decide complexity by 15x

Reply:
Yeah, reusing the test cases :muscle: